experimental/traffic-portal/src/app/core/types/detail/type-detail.component.html

```diff
@@ -16,6 +16,8 @@
 	<tp-loading *ngIf="!type"></tp-loading>
 	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="type">
 		<mat-card-header class="headers-container" *ngIf="!new">
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'cachegroup'" [routerLink]="'/core/cache-groups'" [queryParams]="{search: type.name}">View Cache Groups</a>
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'server'" [routerLink]="'/core/servers'" [queryParams]="{search: type.name}">View Servers</a>
```

Comment:
These should not use search, as that will highlight matches across all string-type properties of the object. For example, if your type name is `EDGE`, then clicking the servers button will show e.g. all cache servers with `edge` in the hostname - which could be quite a lot. And more to the point, inaccurate.

Instead use the appropriate filtering via query parameters e.g. `typeName=EDGE`